pkg/util/oc/pod_commands.go

```diff
@@ -94,10 +94,33 @@ func WaitAllPodsReady(t test.TestHelper, ns string) {
 func WaitCondition(t test.TestHelper, ns string, kind string, name string, condition string) {
 	t.T().Helper()
 	retry.UntilSuccessWithOptions(t, retry.Options().MaxAttempts(30), func(t test.TestHelper) {
-		shell.Executef(t,
-			fmt.Sprintf(`oc wait -n %s %s/%s --for condition=%s  --timeout %s`, ns, kind, name, condition, "10s"),
+		cmd := fmt.Sprintf(`oc wait -n %s %s/%s --for condition=%s  --timeout %s`, ns, kind, name, condition, "10s")
+		shell.Execute(t,
+			cmd,
```

Comment:
Why introduce the `cmd` variable? What's the benefit?

Reply:
Not needed, because before I was using executef. Done